wordpress-mobile/WordPress-Android
# Implement View Site QS task and scroll logic

Fixes #13840 

This PR adds the focus point and snackbar to the "View site" task. The task gets completed if you click on the "View site" button. I've also added scroll logic to the currently active task. Let me know if my approach makes sense, I'm not 100% it's good :-)

To test:
- Make sure the My Site improvements flag is turned on and the app is restarted
- Open a site with QS started
- Click on "View site" 
- Notice the app scrolls to the "View site" item and it's showing a focus point
- Go back to the Quick start card but make sure the Site info block is not visible
- Click on "Upload site icon" or "Update site title" task
- Notice the app scrolls up to the Site info block

PR submission checklist:
